### ISSUE TITLE
[Installation] install OMIScriptProvider conflicts with file from package omi-1.4.0-2.x86_64

### ISSUE THREAD

**Issue by JumpingYang001** [Microsoft/omi-script-provider#35]
Build: 
OMI Build: \\redmond\wsscfs\OSTCData\Builds\omi\develop\1.4.0-2 
Python Provider: \\redmond\wsscfs\OSTCData\Builds\omiscriptprovider\develop\1.1.1-10 

issued ENV:
```
omi64-sles12-t1
omi64-rhel7-t1
omi64-ora6-t1
omi64-ora7-t1
omi64-cent7-t1
```

Error:
```
root@omi64-sles12-t1  # rpm -i /tmp/OMIScriptProvider-*.rpm
        file /opt/omi/bin from install of OMIScriptProvider-1.1.1-10.x86_64 conflicts with file from package omi-1.4.0-2.x86_64
        file /opt/omi/lib from install of OMIScriptProvider-1.1.1-10.x86_64 conflicts with file from package omi-1.4.0-2.x86_64
```

**Comment by alinbalutoiu** [comment]
That is weird, I have tested with OMI version 1.3.0-100 and I encountered the same issue with the RPM package. I wonder how this issue did not show up sooner.

**Comment by JumpingYang001** [comment]
I think this is python provider issue since python provider build is just ready recently.
Looks like OMI version 1.3.0-100 don't support Python provider.
For this issue didn't file before, I need some time to check it, thanks. (Maybe we used rpm -i /tmp/OMIScriptProvider-*.rpm --force to workaround them before, I am not sure, but will check it tomorrow, thanks. )
BTW, I will remove some non-python2.7 from these machines list tomorrow: omi64-sles12-t1, omi64-rhel7-t1,omi64-ora6-t1, omi64-ora7-t1, omi64-cent7-t1.

**Comment by alinbalutoiu** [comment]
OMI version 1.3.0-100 supports the Python provider, if you look in the Build-OMI repo history https://github.com/Microsoft/Build-omi/tree/f328afdd4f010ed4776ac2ff7f9975d070bc8825 you can see that it uses OMI https://github.com/Microsoft/omi/tree/5eacf66d0e684b82822c182b75339378e21a8aa9 which has the commit for the script provider.

Using --force during rpm installation would ignore the file conflict error. There have not been any major changes to the python provider build which would affect this issue.

Anyway, check please a build which includes the PR to see if it fixes the issue on all the systems, thanks.

**Comment by jeffaco** [comment]
This is a bug in the OMI script provider install scripts. Take a look [here](https://github.com/Microsoft/omi-script-provider/blob/master/installbuilder/Base_OMIScriptProvider.data#L21), lines 21 and 22 should be marked as "sysdir" since these directories are NOT owned by OMI script provider.

This is a show stopper bug and must be fixed prior to releasing any packages. Otherwise, customers will have problems during upgrades.

**Comment by JumpingYang001** [comment]
Verified the fix on 1.1.1-12.
